src/applications/travel-pay/components/TravelPayFilters.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VaCheckbox } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+export default function TravelPayFilters(props) {
```

Comment:
I'd love to see a separate component per filter. We can wrap the filter components in this filters component. Thoughts?

Reply:
That's a great idea - I can make a new `TravelPayStatusFilters` component